sdk/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Wed Nov 13 14:53:27 PST 2019
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
Gradle itself is also up to v6.0. You can regenerate the gradle wrapper with:

```
./gradlew wrapper --gradle-version=6.0
```

An exciting new feature (to me anyhow!) is **dependency version alignment**:
> **Aligning versions across multiple dependencies**
> 
> Dependency version alignment allows builds to express that different modules belong to the same logical group (like a platform) and need to have identical (a.k.a aligned) versions in a dependency graph.

Reply:
Gradle 6.0 supports the Android Gradle plugin v3.4 and up, so we should be good to go there. There is one known issue where Gradle 6.0 seems to have broken incremental compilation for Android, but this will be fixed in 6.0.1.

Reply:
`gradle wrapper --gradle-version=6.0.1` worked.